src/doc/src/reference/config.md

```diff
@@ -363,18 +363,19 @@ Can be overridden with the `--target-dir` CLI option.
 ##### `build.rustflags`
 * Type: string or array of strings
 * Default: none
-* Environment: `CARGO_BUILD_RUSTFLAGS` or `RUSTFLAGS`
+* Environment: `CARGO_BUILD_RUSTFLAGS` or `CARGO_ENCODED_RUSTFLAGS` or `RUSTFLAGS`
 
 Extra command-line flags to pass to `rustc`. The value may be a array of
 strings or a space-separated string.
 
 There are three mutually exclusive sources of extra flags. They are checked in
```

Comment:
> three mutually exclusive sources

This should probably say "four" now?

Reply:
Good catch! Thanks! 
https://github.com/rust-lang/cargo/pull/10657